### ISSUE TITLE
Markdown preview scrolling no longer works

### ISSUE THREAD

**Issue by xfzv** [VSCodium/vscodium#694]
**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

**To Reproduce**
1. Delete `$HOME/.config/VSCodium/` to use default settings
2. Start VSCodium
3. Open a markdown file
4. Toggle the preview pane
5. Scroll through either the markdown file or the preview pane
6. The other pane doesn't follow the scrolling

**Expected behavior**
The markdown file and/or the preview pane should follow the scrolling

**Screencast**

https://user-images.githubusercontent.com/78810647/113515734-d902b180-9565-11eb-8d34-6571f8649efc.mp4

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Architecture: x86_64
 - Version: 1.55.0 (installed with [`vscodium-bin`<sup>AUR</sup>](https://aur.archlinux.org/packages/vscodium-bin/))

**Additional context**
It works fine with both:
- `code` installed from Arch Linux community repo
- [`visual-studio-code-bin`<sup>AUR</sup>](https://aur.archlinux.org/packages/visual-studio-code-bin/)
using default settings.

I'm not sure when it stopped working on VSCodium, I haven't used the functionnality for a long time.

I do have the following settings enabled:

```
markdown.preview.scrollEditorWithPreview
markdown.preview.scrollPreviewWithEditor
```

**Comment by lui1s** [comment]
Ey mate. Could you check if you can work with internal links in Markdown? 
Like for example having this: `[link](./file.md)`
Because I'm having trouble with that, the internal links in the preview pane do not work.
External links in my case do work.

**Comment by xfzv** [comment]
> Ey mate. Could you check if you can work with internal links in Markdown?
> Like for example having this: `[link](./file.md)`
> Because I'm having trouble with that, the internal links in the preview pane do not work.
> External links in my case do work.

Same here, local links in the editor pane works but they don't in the preview pane. I didn't try with Microsoft's Visual Studio Code. External links work fine.

I think you should open another issue for this.

**Comment by lui1s** [comment]
I tried on VSCode and it works just fine. As you said local links in the editor mode works but they do not in the preview mode, which is quite annoying because I cant navigate over my files.
@xfzv I'm pretty new to github, could you open the issue, and refer to these messages?

**Comment by keremispirli** [comment]
I can confirm that scrolling markdown editor with preview and vice versa do not work on Windows 10, either.

**Comment by rafaelmaeuer** [comment]
I have the same issue running VSCodium on macOS 11.3. Scroll-Sync stopped working with on of the recent updates...

**Comment by RotationMatrix** [comment]
Confirmed on macOS 10.14.6 with 1.56.2. This feature does work with 1.54.3 on the same OS though.

**Comment by AkshayChn** [comment]
Same issue with 1.56.2 Appimage.
OS: Ubuntu 20.04
Release File: `VSCodium-1.56.2-1620951495.glibc2.16-x86_64.AppImage`

**Comment by daiyam** [comment]
Confirmed. It's due to the error `Error: Unable to read file '/Applications/VSCodium.app/Contents/Resources/app/extensions/markdown-language-features/media/pre.js'`.
I will look into it.

**Comment by daiyam** [comment]
The fix has been merged so the issue should be fixed in the next release.

**Comment by daiyam** [comment]
It's has been fixed in `1.58.0`
